### PR TITLE
Esbuild pre-requisites

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by martinpitt — APPROVED** [review]

**Review by martinpitt — APPROVED** [review]

**Comment by KKoukiou** [comment]
@martinpitt there is something wrong with two files and css https://cockpit-logs.us-east-1.linodeobjects.com/pull-18436-20230302-151917-395b1c7d-fedora-37/log.html

**Comment by KKoukiou** [comment]
@martinpitt tests look now good. Can you please take onother look :) ?

File: pkg/systemd/overview.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@use "page";
```
Comment (martinpitt) [inline pkg/systemd/overview.scss:1]: OOI, does that make a difference? the next line imports system-global.scss, which imports page.scss as the first thing. I'm worried about duplicating CSS with such duplicate imports.

**Review by martinpitt — COMMENTED** [review]
Nice work figuring out the specificity bug! I have one question. I retriggered the queued tests which apparently got stuck.

File: pkg/systemd/overview.scss (hunk 1 above)
Comment (KKoukiou) [reply pkg/systemd/overview.scss:1]: Oh probably it's not needed. Good catch.

Comment (martinpitt) [reply pkg/systemd/overview.scss:1]: I'm fine with leaving it in if the generated .css is the same in both cases. In other words, if dart-sass compiler de-duplicates imports.

**Review by martinpitt — DISMISSED** [review]

**Review by martinpitt — APPROVED** [review]

**Comment by martinpitt** [comment]
This is one of these "impossible to get green" PRs. I went through the failures, and I'm fairly convinced that they are unrelated and random. Most failures are only in the third retry, or due to some unexpected messages in the pybridge (which we already know about).

**Pull request merged**